prime-dotnet-webapi/Controllers/ProvisionerAccessController.cs

```diff
@@ -136,6 +136,19 @@ public async Task<ActionResult> GetGpid()
             return Ok(await _enrolleeService.GetActiveGpidAsync(User.GetPrimeUserId()));
         }
 
+        // GET: api/provisioner-access/gpid-detail
+        /// <summary>
+        /// Gets the GPID and detail info for the user. Only a valid token is required, no role is required.
+        /// </summary>
+        [HttpGet("gpid-detail", Name = nameof(GetGpidDetail))]
+        [Authorize]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(typeof(ApiResultResponse<string>), StatusCodes.Status200OK)]
```

Comment:
I think this is only meta-data for Swagger but should that be `[ProducesResponseType(typeof(ApiResultResponse<HpdidLookup>), StatusCodes.Status200OK)]`?

Reply:
Good catch. it is fixed.